src/components/form-elements/Textarea.jsx

```diff
@@ -19,6 +19,7 @@ const Textarea = props => {
     autoHeight,
     value,
     className,
+    Type = 'textarea',
```

Comment:
shouldn't this be in some kind of object, like enum or something?

Reply:
textarea it's default and we basically want only textarea or React class looking like textarea. Ex. react-textarea-autosize from npm :)